builtin/providers/google/resource_container_cluster.go

```diff
@@ -282,6 +288,24 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		cluster.InitialClusterVersion = v.(string)
 	}
 
+	if v, ok := d.GetOk("additional_zones"); ok {
```

Comment:
Is there a way to update this without recreating the whole cluster? I did not find any. I guess I should set `ForeNew: True` then. Currently, the only argument that can be updated is `node_version`.